app/src/main/res/xml/preferences_server.xml

```diff
@@ -36,6 +36,12 @@
             android:key="connection_internal"
             android:title="@string/pref_connection_internal"
             app:useSimpleSummaryProvider="true"/>
+        <Preference
+            app:isPreferenceVisible="false"
+            android:key="connection_security_level"
+            android:icon="@drawable/ic_lock"
+            android:title="@string/connection_security_level_title"
+            android:summary="@string/connection_security_level_default_summary"/>
```

Comment:
Don't set the default summary to a long string to avoid the screen moving around immediately when opened due to the summary changing.

Reply:
I'm going to keep it blank then, it is going to be set programmatically only